dash/webapp/next-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
If this file is empty do we need it at all? Or do the comments do something?

Reply:
It was automatically created when installing packages, we have the same in `content` and `catalogue`.